src/network/asyncore_pollchoose.py

```diff
@@ -50,16 +45,17 @@
 many of the difficult problems for you, making the task of building
 sophisticated high-performance network servers and clients a snap.
 """
```

Comment:
Why do you want to include license into the doc?

Reply:
I don't know about this, that's why I didn't remove. Should I remove that?